section/include/moho.h

```diff
@@ -1455,4 +1474,18 @@ struct WRenViewport // : WD3DViewport
 	Clutter clutter;
 	// at 0x2134
 	Silhouette silhouette;
-};
\ No newline at end of file
+};
+
+namespace incomplete {
+	struct Command
+	{
+		Command* self;
+		uint8_t pad1[0x1C];
+		mohostring* blueprintId; // at 0x20 // valid if != nullptr
+		uint8_t pad2[0x34];
+		int commandType; // at 0x58 // see EUnitCommandType; sub_552170
+		uint8_t pad3[0x4];
+		int targetId; // at 0x60 // valid if != 0xF0000000
+		Vector3f pos; // at 0x64

```

Comment:
[CUnitCommand](https://github.com/FAForever/FA-Binary-Patches/blob/master/section/include/moho.h#L748-L754)

Reply:
Could you elaborate?